types/genesis.go

```diff
@@ -26,17 +26,7 @@ type GenesisDoc struct {
 	ConsensusParams *ConsensusParams   `json:"consensus_params,omitempty"`
 	Validators      []GenesisValidator `json:"validators"`
 	AppHash         cmn.HexBytes       `json:"app_hash"`
-	AppStateJSON    json.RawMessage    `json:"app_state,omitempty"`
-	AppOptions      json.RawMessage    `json:"app_options,omitempty"` // DEPRECATED
-}
-
-// AppState returns raw application state.
-// TODO: replace with AppState field during next breaking release (0.18)
-func (genDoc *GenesisDoc) AppState() json.RawMessage {
-	if len(genDoc.AppOptions) > 0 {
-		return genDoc.AppOptions
-	}
-	return genDoc.AppStateJSON
+	AppState        json.RawMessage    `json:"app_state,omitempty"`
```

Comment:
Technically breaking on Go APIs but preserves the JSON. According to our latest SemVer, that's a non breaking version change. Cool.